ecommerce/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+from ecommerce.models import Product
+
+
+class ProductAdmin(admin.ModelAdmin):
```

Comment:
Could you update this to use django-reversion's admin?

https://django-reversion.readthedocs.io/en/stable/admin.html